docs/Developer Reference/link-token-contracts.md

```diff
@@ -92,7 +90,7 @@ LINK is native to Ethereum, so to use LINK on other chains, it must be bridged.
 
 > 🚰BSC Faucet
 >
-> Testnet LINK is available from <a href="https://linkfaucet.protofire.io/bsctest" target="_blank" rel="noreferrer, noopener">the community faucet</a>.
+> Testnet LINK is available from <a href="https://faucets.chain.link/chapel" target="_blank" rel="noreferrer, noopener">the community faucet</a>.
```

Comment:
I don't think these are community faucets any more - maybe the text should match what we say above for kovan/rinkeby?